isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -421,7 +421,14 @@ namespace Isis{
           }
 
           bandGeom->collect(*cam, *incube, doGeometry, doPolygon, getFootBlob, precision);
-
+         
+          // When getFootBlob is true, incube is closed in the above function call
```

Comment:
I'm not sure why the collect call above closes a cube that was passed in to it. Seems like a better solution would be to remove the close inside collect() and then close it in at the bottom of this function caminfo(cube, ui) or let what ever code opened it do the closing (i.e., caminfo(ui)).